src/main/java/com/revature/controllers/UserController.java

```diff
@@ -43,6 +45,9 @@
 	@Autowired
 	private DistanceService distanceService;
 
+	@Autowired
+	private Validator validator;
```

Comment:
Does this validator variable not need to be used anywhere?

Reply:
fixed in 93def6a